ui/src/views/Project.vue

```diff
@@ -59,5 +71,27 @@ export default class ProjectView extends Vue {
       }
     }
   }
+
+  copyProjectURI () {
+    const inputComponent = this.$refs.uriInput as Vue
+    const inputElement = inputComponent?.$el.querySelector('input')
+
+    if (!inputElement) { return }
+
+    inputElement.select()
+    const success = document.execCommand('copy')
```

Comment:
MDN says it's an obsolete method. Is there a better way nowadays to copy?

Reply:
That seems more complicated than I thought: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard

It's weird, they don't mention anything about it being obsolete in that article. And they specifically say the the new Clipboard API is not implemented everywhere (confirmed by https://caniuse.com/#search=clipboard)...

I'll do a bit more research.

Reply:
I think `document.execCommand('copy')` is still the safest way to copy to clipboard (without adding a dependency).